packages/patternfly-4/react-docs/src/html.js

```diff
@@ -0,0 +1,39 @@
+import React from "react"
+import PropTypes from "prop-types"
+
+export default function HTML(props) {
+  return (
+    <html {...props.htmlAttributes}>
+      <head>
+        <meta charSet="utf-8" />
+        <meta httpEquiv="x-ua-compatible" content="ie=edge" />
+        <meta
+          name="viewport"
+          content="width=device-width, initial-scale=1, shrink-to-fit=no"
+        />
+        {props.headComponents}
+      </head>
+      <body {...props.bodyAttributes} className="pf-m-redhat-font">
```

Comment:
We are using react-helmet in other places to alleviate the need for this - you can add this instead (only the body element needs to be added, other code added for reference so you can find it):

in react-docs/src/templates/sidebarLayout.js 
```
<Helmet>
  <html lang="en-US" />
  <meta charSet="utf-8" />
  <meta name="description" content="PatternFly React Documentation" />
  <meta name="keywords" content="React, PatternFly, Red Hat" />
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <body className='pf-m-redhat-font' />
</Helmet>
```

Reply:
I followed these instructions from the Gatsby documentations.  Are you saying that I do not need to copy the html.js file?
https://www.gatsbyjs.org/docs/custom-html/

Reply:
No - the html.js file isn't needed for just adding a class on the body tag, react-helmet supports that with a single line change to sidebarLayout